templates/catalog/_partials/miniatures/product.tpl

```diff
@@ -185,9 +185,9 @@
                 <div class="quantity-button js-quantity-button w-100 w-sm-auto">
                   {include file='components/qty-input.tpl'
                     attributes=[
-                      "id"=>"quantity_wanted_{$product.id_product}",
-                      "value"=>"{if $product.cart_quantity && $product.cart_quantity >= $product.minimal_quantity}1{else}{$product.minimal_quantity}{/if}",
-                      "min"=>"{if $product.cart_quantity && $product.cart_quantity >= $product.minimal_quantity}1{else}{$product.minimal_quantity}{/if}"
+                      "id" => "quantity_wanted_{$product.id_product}",
+                      "value" => "1",
```

Comment:
I'm not sure why you fixed the value to 1 instead of using `$product.minimal_quantity`. The same in other file.

Reply:
For this, it's an issue with the data in cache before template generation.
The `cart_quantity`, that must be unique by users, is in cache too and for everybody! With your solution, if we do a `cache:clear` at any time, it's working well.
So yeah, it's a core issue, but force to 1 and add some javascript can do the trick as long as we fix this cache issue in the core.

Reply:
In all cases, this isn't a fault to cache the product list 🤷‍♂️
But the `cart_quantity` don't be cached in a global state.

Reply:
Could `{nocache}` help here?
https://www.smarty.net/docs/en/language.function.nocache.tpl

Reply:
Unfortunately, it's seems not enough.

Reply:
I understand the cache problem, but then shouldn't the default value be `{$product.minimal_quantity}` It doesn't seem logic to have a value always to 1 when the min attribute under can be higher no?

It would mean the default cached page would use the min quantity for everyone and then your JS update handles updating/reducing it when relevant no?